force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -214,8 +266,13 @@ public with sharing class ClassesObjectsMethods {
     public static Integer calculateYearDifference(Date d1, Date d2) {
 
         // Subtract the difference in years
-
-        return null; // Replace null with the variable you used to store the result
+        if (d1 == null || d2 == null) {
+            return null;
+        } 
+            
+        Integer numOfYears = d2.year() - d1.year();
```

Comment:
This is great and will get you the difference but if you want to get a positive number every time, you can try the Math.abs() method.